tck/src/test/java/org/reactivestreams/tck/RangePublisherTest.java

```diff
@@ -0,0 +1,176 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams.tck;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.*;
+
+import org.reactivestreams.*;
+import org.testng.annotations.*;
+
+@Test
+public class RangePublisherTest extends PublisherVerification<Integer> {
+
+    static final Map<Integer, StackTraceElement[]> stacks = new ConcurrentHashMap<Integer, StackTraceElement[]>();
+    
+    static final Map<Integer, Boolean> states = new ConcurrentHashMap<Integer, Boolean>();
+    
+    static final AtomicInteger id = new AtomicInteger();
+    
+    @AfterClass
+    public static void afterClass() {
+        boolean fail = false;
+        StringBuilder b = new StringBuilder();
+        for (Map.Entry<Integer, Boolean> t : states.entrySet()) {
+            if (!t.getValue()) {
+                b.append("\r\n-------------------------------");
+                for (Object o : stacks.get(t.getKey())) {
+                    b.append("\r\nat ").append(o);
+                }
+                fail = true;
+            }
+        }
+        if (fail) {
+            throw new AssertionError("Cancellations were missing:" + b);
+        }
+    }
+    
+    public RangePublisherTest() {
+        super(new TestEnvironment(25));
```

Comment:
This isn't the Rx suite nor does it impact the Rx suite, so that does not really matter :-) 

It does, however, break the configurability of the timeouts for how we use them in this repo: since the setting is given via env value: https://github.com/reactive-streams/reactive-streams-jvm/blob/master/.travis.yml#L14 and settings in code win over settings from env (documented here: https://github.com/reactive-streams/reactive-streams-jvm/tree/master/tck#timeout-configuration ).

Please note that timeout does not hurt suite time all that much, the more painful one is  (as explained here: https://github.com/reactive-streams/reactive-streams-jvm/tree/master/tck#timeout-configuration
 ) contributes much more to "actually waiting doing nothing time", so if anything the other one is more influential AFAIR.

Please stick to the default here so the `.travis.yml` configuration continues to work in the project, thanks.